src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -704,6 +703,7 @@ void ImageReplayer<I>::on_stop_local_image_close_finish(int r)
     std::swap(m_on_stop_finish, on_finish);
   }
 
+  update_mirror_image_status(true, STATE_STOPPED);
```

Comment:
At this stage m_state_desc is not cleared (it is cleared in handle_stop), so I observe after stop:

```
zhuzha:/tmp/tmp.rbd_mirror% ceph --admin-daemon /tmp/tmp.rbd_mirror/rbd-mirror.cluster1_daemon.cluster1.asok rbd mirror stop
zhuzha:/tmp/tmp.rbd_mirror% rbd --cluster cluster1 -p mirror mirror pool status --verbose
health: OK
images: 2 total
    2 stopped

test:
  global_id:   8490718b-edf5-4e7a-89d0-1760e7171b6b
  state:       up+stopped
  description: bootstrapping, CLOSE_REMOTE_IMAGE
  last_update: 2016-05-18 14:41:03

test1:
  global_id:   0acf3ba3-7305-4405-9efd-872246344624
  state:       up+stopped
  description: bootstrapping, CLOSE_REMOTE_IMAGE
  last_update: 2016-05-18 14:41:03
```